components/automate-ui/src/app/pages/automate-settings/automate-settings.component.html

```diff
@@ -141,10 +141,10 @@ <h2>Client Runs</h2>
               <input chefInput 
                 type="number" 
                 min="0" 
-                formControlName="threshold" 
+                formControlName="threshold" [resetOrigin]="shouldResetValues"
                 (keydown)="preventNegatives($event.key)" />
```

Comment:
Suggest rewriting some of the elements so they appear consistent.
1. This `<input>` element is the most egregious example--it puts one attribute per line. All the other ones are on a single line. 
2. Put the attributes in the same order. It just helps lessen the mental effort needed to read the file. For instance, here are the first two in the file where things are in a different order:
```
<input chefInput formControlName="threshold" [resetOrigin]="shouldResetValues" type="number" min="0" 
<input chefInput type="number" min="0" formControlName="threshold" [resetOrigin]="shouldResetValues"
```